From dc244a7dce4b21d8f7fb67f7c08cd5d25c0af818 Mon Sep 17 00:00:00 2001
From: Dongli Zhang <dongli.zhang@oracle.com>
Date: Thu, 21 Mar 2019 09:09:09 +0200
Subject: [PATCH 1/4] blk-mq: Rmove unused 'nr_expired' from blk_mq_hw_ctx

There is no usage of 'nr_expired'.

The 'nr_expired' was introduced by commit 1d9bd5161ba3 ("blk-mq:
replace timeout synchronization with a RCU and generation based
scheme"). Its usage was removed since commit 12f5b9314545 ("blk-mq:
Remove generation seqeunce").

Signed-off-by: Dongli Zhang <dongli.zhang@oracle.com>
---
 include/linux/blk-mq.h | 1 -
 1 file changed, 1 deletion(-)

diff --git a/include/linux/blk-mq.h b/include/linux/blk-mq.h
index 6a514e5136f4..7ce42e7a127c 100644
--- a/include/linux/blk-mq.h
+++ b/include/linux/blk-mq.h
@@ -57,7 +57,6 @@ struct blk_mq_hw_ctx {
 	unsigned int		queue_num;
 
 	atomic_t		nr_active;
-	unsigned int		nr_expired;
 
 	struct hlist_node	cpuhp_dead;
 	struct kobject		kobj;
-- 
2.21.0.157.g0e94f7aa73

